one_fm/one_fm/custom/issue.json

```diff
@@ -239,9 +239,138 @@
    "translatable": 1,
    "unique": 0,
    "width": null
+  },
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2022-03-24 10:49:35.649533",
+   "default": null,
+   "depends_on": "eval:doc.communication_medium == 'WhatsApp'",
+   "description": null,
+   "docstatus": 0,
+   "dt": "Issue",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "whatsapp_number",
+   "fieldtype": "Data",
+   "hidden": 0,
+   "hide_border": 0,
+   "hide_days": 0,
+   "hide_seconds": 0,
+   "idx": 6,
+   "ignore_user_permissions": 0,
+   "ignore_xss_filter": 0,
+   "in_global_search": 0,
+   "in_list_view": 0,
+   "in_preview": 0,
+   "in_standard_filter": 0,
+   "insert_after": "communication_medium",
+   "label": "WhatsApp Number",
+   "length": 0,
+   "mandatory_depends_on": "eval:doc.communication_medium == 'WhatsApp'",
+   "modified": "2022-03-24 10:49:35.649533",
+   "modified_by": "Administrator",
+   "name": "Issue-whatsapp_number",
+   "no_copy": 0,
+   "non_negative": 0,
+   "options": null,
+   "owner": "Administrator",
+   "parent": null,
+   "parentfield": null,
+   "parenttype": null,
+   "permlevel": 0,
+   "precision": "",
+   "print_hide": 0,
+   "print_hide_if_no_value": 0,
+   "print_width": null,
+   "read_only": 1,
+   "read_only_depends_on": null,
+   "report_hide": 0,
+   "reqd": 0,
+   "search_index": 0,
+   "translatable": 1,
+   "unique": 0,
+   "width": null
+  }
+ ],
+ "custom_perms": [
```

Comment:
We dont need the custom perms.

Reply:
I would not know where the permission came from I may have to review it.